test/spec/collection.test.js

```diff
@@ -206,13 +212,15 @@ describe("collection", () => {
         [count]: 2
       });
 
-      expect(spec.conform([1, 2]), "array - count, happy").to.deep.equal([
-        1,
-        2
-      ]);
-      expect(spec.conform(new Set([1, 2])), "set - count, happy").to.deep.equal(
-        new Set([1, 2])
-      );
+      expect(
+        spec.conform([1, 2]),
+        "array - count, happy"
+      ).to.contain.ordered.members([1, 2]);
+
+      expect(spec.conform(new Set([1, 2])), "set - count, happy").to.be
+        .a("Set")
+        .that.contains(1, 2)
```

Comment:
`deep.equal` failed for Sets. It considered the two Sets equal, even though one has 3 elements and the other has 2.